pkg/pgmodel/ingestor/inserter.go

```diff
@@ -135,7 +135,6 @@ func (p *pgxInserter) InsertData(rows map[string][]model.SamplesInfo) (uint64, e
 		workFinished.Wait()
 		select {
 		case err = <-errChan:
-			fmt.Println("here", err)
```

Comment:
I know "here" is not proper, but is the error logged somewhere else?

Reply:
Yes, we are already logging it. https://github.com/timescale/promscale/blob/724c48f858c92e3d319f7e1ad1ff0870e9abf8ed/pkg/api/write.go#L74